canvas/models/models.py

```diff
@@ -274,7 +274,7 @@ def has_solved_event(self, user):
 
         uqjs = UserQuestionJunction.objects.filter(user_id=user.id, is_solved=True, question__in=event_questions)
 
-        if uqjs.exists():
+        if uqjs.count() == event_questions.count():
             return True
         return False
```

Comment:
I'm not sure if there is a better way to do this. 
```
if xxx:
   return True
return False
```
feels not as clean. Opening to advice!

Reply:
I think this code is fine tbh, if I was to nitpick I suppose you could just `return uqjs.count() == event_questions.count()` instead of the conditional and it would logically be the same though